android-components/gradle.properties

```diff
@@ -9,7 +9,7 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseParallelGC
```

Comment:
I think we should be keeping the `UseParallelGC`. Seems useful to have it in CI from what I've read so far:

> The parallel collector (also referred to here as the throughput collector) is a generational collector similar to the serial collector; the primary difference is that multiple threads are used to speed up garbage collection.

https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/parallel.html

> The parallel scavenge collector (Enabled using -XX:UseParallelGC). This is like the previous parallel copying collector, but the algorithm is tuned for gigabyte heaps (over 10GB) on multi-CPU machines. This collection algorithm is designed to maximize throughput while minimizing pauses.

https://www.javaperformancetuning.com/news/qotm026.shtml

Reply:
Ah the `HeapDumpOnOutOfMemoryError` is also what helps us know when CI runs out of memory (like in the situations that we're currently seeing).

> The -XX:+HeapDumpOnOutOfMemoryError command-line option tells the HotSpot VM to generate a heap dump when an allocation from the Java heap or the permanent generation cannot be satisfied. There is no overhead in running with this option, and so it can be useful for production systems where OutOfMemoryError takes a long time to surface.

https://www.oracle.com/java/technologies/javase/clopts.html#gbzrr

Reply:
So probably?

```
org.gradle.jvmargs=-Xmx7g -Xms2g -XX:MaxMetaspaceSize=6g -XX:+HeapDumpOnOutOfMemoryError -XX:+UseParallelGC
```

Reply:
I feel like we dropped those for Fenix because they were the defaults already anyway.

Reply:
I hope so! If it's the default, then there is no harm in keeping them for now.